Cargo.toml

```diff
@@ -30,7 +30,7 @@ homepage = "https://github.com/uuid-rs/uuid"
 name = "uuid"
 readme = "README.md"
 repository = "https://github.com/uuid-rs/uuid"
-version = "0.8.1" # remember to update html_root_url in lib.rs
+version = "0.8.2" # remember to update html_root_url in lib.rs
```

Comment:
Leave this unchanged. We use a separate PR to handle the version update just before release

Reply:
Reverted. Thanks.